packages/devtools_app/test/shared/common_widgets_test.dart

```diff
@@ -211,4 +211,53 @@ void main() {
       expect(findCheckboxValue(), isTrue);
     });
   });
+
+  group('AreaPaneHeader', () {
+    const titleText = 'The title';
+    const centerActionText = 'The Center Action';
+    const centerActionContainerKey = Key('scrollableCenterActionsContainer');
+    const centerAction = Text(centerActionText);
+
+    setUp(() {
+      setGlobal(IdeTheme, IdeTheme());
+    });
```

Comment:
instead of doing this here, call `setGlobal(IdeTheme, IdeTheme());` in the top level main method. You can get rid of the other call at the top of the file too when you do this. You can also move `setGlobal(ServiceConnectionManager, FakeServiceManager());` to be called in top level main.